packages/core/src/build/config/config.ts

```diff
@@ -10,14 +11,18 @@ import {
 } from "@/config/networks.js";
 import { chains } from "@/utils/chains.js";
 import { toLowerCase } from "@/utils/lowercase.js";
+import { createRequestQueue } from "@/utils/requestQueue.js";
 import type {
   Factory,
   LogFilter,
   Source,
   Topics,
 } from "../../config/sources.js";
 
-export async function buildNetworksAndSources({ config }: { config: Config }) {
+export async function buildNetworksAndSources({
+  config,
+  common,
```

Comment:
Don't have a great solution, but it's smelly to me that this function now needs `common`/`metrics`.